recipes/pysm3/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "pysm3" %}
+{% set version = "3.1.dev238" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 596c375b353f1fa051d559dfb8c896377aaf886ba34326aa0f7f4f5ac83b4801
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - healpy
+    - toml
+    - numba
+    - scipy
+    - astropy-helpers
```

Comment:
Do you actually need all of these deps to be able to install?  Most of these seem like run deps

Reply:
it looks like they are necessary, I tried removing them and gave importerror on `numpy`